splitio/client/factory.py

```diff
@@ -453,9 +547,13 @@ def _build_localhost_factory(cfg):
 def get_factory(api_key, **kwargs):
     """Build and return the appropriate factory."""
     try:
+        active_factory_count = 0
+        redundant_factory_count = 0
```

Comment:
That is what the code is doing, these variables are updated inside the lock block every time a new factory instance is created

Reply:
those variables are scoped to the get_factory function. 

every time the function runs, the variables are set to 0. 
if this apikey has already been used, on line 555 & 556 you increment them both (now they're 1).
otherwise you only increment the active_Factory_count (now 1).

if you previously had 220 factories for the same apikey, the config will report 1 for active, and 1 for redundant.

active factories should be fetched from `len(_INSTANTIATED_FACTORIES)`
redundant factories for this apikey should be fetched from `_INSTANTIATED_FACTORIES[api_key]`

Reply:
Fixed